### DISCUSSION TITLE
MacYTDL not working on Catalina

### DISCUSSION THREAD

**Discussion by avcodex** [section83/MacYTDL#73] [Q&A]
Hello,

I am trying to run this app on Catalina 10.15.7 but its freezing the application when I try to download.
I try to Uninstall in utilities but it does not look it completes anything.
How can I uninstall MacYTDL and all the extra packages ?
Thanks

**Comment by section83** [comment]
Apologies for all the trouble you've had.  I did test MacYTDL on Catalina some time ago and it worked normally.  It definitely still works on Mojave.  I've not see MacYTDL freezing during a download.  The main MacYTDL dialog should be redisplayed after the download starts.  Did the main dialog redisplay ?  MacYTDL might "hang" if it's trying to do something and can't get permission or access to something.  However, normally, it should timeout and report an error.  One thing you could try is to make sure you try MacYTDL in the user account which copied MacYTDL from the DMG to Applications.

After starting the download, did you see a "monitor" window ? It would look like this:

<img width="281" alt="Monitor -  Progressing" src="https://github.com/section83/MacYTDL/assets/7471595/2d205e81-79d4-49af-9dd9-071079ae75cc">

I can't think of anything else at present. If I get another idea I will add a post to this discussion.

Anyway, the Uninstall function just moves everything to the Bin.  If you want to check that everything has been deleted, here is a list:

- MacYTDL applet – did you copy it to Applications ?
- All preferences and logs: [Your Mac]/Users/[your home]/Library/Preferences/MacYTDL
- Libraries:
  - [Your Mac]/Users/[your home]/Library/Script Libraries/DialogToolkitMacYTDL.scptd
  - [Your Mac]/Users/[your home]/Library/Script Libraries/Myriad Tables Lib.scptd [You might not have installed this]
- Service: [Your Mac]/Users/[your home]/Library/Services/Send-URL-To-MacYTDL.workflow   [You might not have installed this]
- Binaries:
  - [Your Mac]/usr/local/bin/yt-dlp
  - [Your Mac]/usr/local/bin/ffmpeg
  - [Your Mac]/usr/local/bin/ffprobe
  - [Your Mac/usr.local/bin/AtomicParsley  [You might not have installed this]

You will need to provide credentials of an administrator account on you Mac to delete some of those files.

Many thanks for making contact.
